lib/src/constants/pkg.dart

```diff
@@ -38,6 +38,9 @@ const String assetsPath = 'assets';
 const String mediaPath = 'assets/media';
 const String surveyAsset = 'assets/surveys.md';
 
+const String defaultFileName = 'result.json';
```

Comment:
Perhaps call it `survey.json`

Reply:
Actually, it is not always a survey! Is it possible to default the filename to be the menu item name. In the example this will be `survey_1.json` or `survey_2.json` or `survey_3.json` depending which survey.

Reply:
> Actually, it is not always a survey! Is it possible to default the filename to be the menu item name. In the example this will be `survey_1.json` or `survey_2.json` or `survey_3.json` depending which survey. The name is derived from the menu item/survey name and normalised for a nice filename (lowercase and replace space with underscore).

Yes, it is a very good idea. I'll implement it.